flink-connectors/flink-connector-elasticsearch6/pom.xml

```diff
@@ -174,95 +174,6 @@ under the License.
-						<executions>
-							<execution>
-								<phase>package</phase>
-								<goals>
-									<goal>shade</goal>
-								</goals>
-								<configuration>
-									<shadedArtifactAttached>true</shadedArtifactAttached>
-									<shadedClassifierName>sql-jar</shadedClassifierName>
-									<filters>
-										<filter>
-											<artifact>*:*</artifact>
-											<!-- It is difficult to find out artifacts that are really required by ES. -->
-											<!-- We use hard filters for now to clean up the SQL JAR. -->
-											<excludes>
-												<exclude>com/carrotsearch/**</exclude>
-												<exclude>com/sun/**</exclude>
-												<exclude>com/tdunning/**</exclude>
-												<exclude>config/**</exclude>
-												<exclude>forbidden/**</exclude>
-												<exclude>joptsimple/**</exclude>
-												<exclude>META-INF/services/com.fasterxml.**</exclude>
-												<exclude>META-INF/services/org.apache.lucene.**</exclude>
-												<exclude>META-INF/services/org.elasticsearch.**</exclude>
-												<exclude>META-INF/versions/**</exclude>
-												<exclude>modules.txt</exclude>
-												<exclude>mozilla/**</exclude>
-												<exclude>org/HdrHistogram/**</exclude>
-												<exclude>org/joda/**</exclude>
-												<exclude>org/tartarus/**</exclude>
-												<exclude>org/yaml/**</exclude>
-												<exclude>plugins.txt</exclude>
-											</excludes>
-										</filter>
-									</filters>
-									<relocations>
-										<!-- Force relocation of all Elasticsearch dependencies. -->
-										<relocation>
-											<pattern>org.apache.commons</pattern>
-											<shadedPattern>org.apache.flink.elasticsearch6.shaded.org.apache.commons</shadedPattern>
-										</relocation>
-										<relocation>
-											<pattern>org.apache.http</pattern>
-											<shadedPattern>org.apache.flink.elasticsearch6.shaded.org.apache.http</shadedPattern>
-										</relocation>
-										<relocation>
-											<pattern>org.apache.lucene</pattern>
-											<shadedPattern>org.apache.flink.elasticsearch6.shaded.org.apache.lucene</shadedPattern>
-										</relocation>
-										<relocation>
-											<pattern>org.elasticsearch</pattern>
-											<shadedPattern>org.apache.flink.elasticsearch6.shaded.org.elasticsearch</shadedPattern>
-										</relocation>
-										<relocation>
-											<pattern>org.apache.logging</pattern>
-											<shadedPattern>org.apache.flink.elasticsearch6.shaded.org.apache.logging</shadedPattern>
-										</relocation>
-										<relocation>
-											<pattern>com.fasterxml.jackson</pattern>
-											<shadedPattern>org.apache.flink.elasticsearch6.shaded.com.fasterxml.jackson</shadedPattern>
-										</relocation>
-									</relocations>
-									<!-- Relocate the table format factory service file. -->
-									<transformers>
-										<transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
```

Comment:
Why can we delete this transformer in the `flink-sql-connector-elasticsearch6` module?

Reply:
pulled in through the `shade-flink` execution